### DISCUSSION TITLE
Update Patreon link to enable contributions

### DISCUSSION THREAD

**Discussion by protobi-pieter** [dolanmiu/docx#1455] [General]
This is a very well designed and documented library that merits support.  But the Patreon link on the README.md https://www.patreon.com/dolanmiu/creators doesn't appear to be configured to accept contributions.  Normally I'd expect to see one or more choices under “Select a membership level”.   Is there a more current way to contribute something other than code?

**Comment by dolanmiu** [comment]
Done! added
